shell/platform/android/test/io/flutter/embedding/engine/systemchannels/PlatformChannelTest.java

```diff
@@ -26,18 +26,53 @@
 @RunWith(RobolectricTestRunner.class)
 public class PlatformChannelTest {
     @Test
-    public void setSystemExclusionRectsSendsSuccessMessageToFramework() throws JSONException {
+    public void itSendsSuccessMessageToFrameworkWhenSettingSystemGestureExclusionRects() throws JSONException {
+        // --- Test Setup ---
         DartExecutor dartExecutor = mock(DartExecutor.class);
         PlatformChannel platformChannel = new PlatformChannel(dartExecutor);
         PlatformMessageHandler platformMessageHandler = mock(PlatformMessageHandler.class);
         platformChannel.setPlatformMessageHandler(platformMessageHandler);
+        Result result = mock(Result.class);
+
+        int top = 0;
```

Comment:
Let's avoid placing the values in variables. I think it only makes it more difficult to see what's going into the rect. Generally we hard code input and output in tests to eliminate all possible logic in tests.